app/src/main/res/layout/activity_multimeter.xml

```diff
@@ -18,7 +18,7 @@
             android:layout_height="?attr/actionBarSize"
             android:background="?attr/colorPrimary"
             app:popupTheme="@style/AppTheme.PopupOverlay"
-            app:title="Multimeter" />
+            app:title="@string/multimeter" />
```

Comment:
Similarly here. Please double check if the resource is there and try running the updated code in your PC to see if it's working before making a PR.